src/itemrenderer.cpp

```diff
@@ -144,7 +145,7 @@ void render_html(
 
 void item_renderer::render_plaintext(
 	const std::string& source,
-	std::vector<std::pair<LineType, std::string>>& lines)
+	std::vector<std::pair<LineType, std::string>>& lines, bool raw)
```

Comment:
I think `to_plain_text()` should pass a `true` for this new parameter.

Reply:
Good catch, fixed in https://github.com/newsboat/newsboat/pull/1595/commits/799baac5d8390fff5b129c138b1c80b1d562ee60